packages/adapter-static/test/utils.js

```diff
@@ -1,122 +1,46 @@
-import { execSync } from 'node:child_process';
-import fs from 'node:fs';
-import http from 'node:http';
-import { fileURLToPath } from 'node:url';
-import sirv from 'sirv';
-import { chromium, webkit, firefox } from 'playwright';
-import * as uvu from 'uvu';
+import { test as base, devices } from '@playwright/test';
 
-const known_browsers = {
-	chromium: chromium,
-	firefox: firefox,
-	webkit: webkit
+const known_devices = {
+	chromium: devices['Desktop Chrome'],
+	firefox: devices['Desktop Firefox'],
+	webkit: devices['Desktop Safari']
 };
-const test_browser_name = /** @type {keyof typeof } */ (process.env.KIT_E2E_BROWSER ?? 'chromium');
+const test_browser = /** @type {keyof typeof known_devices} */ (
+	process.env.KIT_E2E_BROWSER ?? 'chromium'
+);
```

Comment:
Looks like these tests aren't run cross-platform. For adapters I think the only cross-platform thing that needs to be tested would be dealing with Windows' path oddities.

Should we remove this part altogether or make the tests run cross-platform?

Reply:
ok. I went ahead and removed it for now